packages/block-library/src/preformatted/transforms.js

```diff
@@ -41,6 +41,15 @@ const transforms = {
 			blocks: [ 'core/code' ],
 			transform: ( attributes ) => createBlock( 'core/code', attributes ),
 		},
+		{
+			type: 'block',
+			blocks: [ 'core/verse' ],
+			transform: ( { content, anchor } ) =>
+				createBlock( 'core/verse', {
+					content,
+					anchor,
+				} ),
```

Comment:
```suggestion
			transform: ( attributes ) =>
				createBlock( 'core/verse', attributes ),
```

Are there any problems if we use the code changes above?

Reply:
I've updated the code with the suggested changes.